metadata/experiment.go

```diff
@@ -50,6 +50,8 @@ type ExperimentDefinition struct {
 	ClusterDefinition ClusterDefinition
 
 	ScenarioDefinition ScenarioDefinition
+
+	TrialDefinition TrialDefinition
```

Comment:
We should just have `Trials []TrialDefinition` here.

Reply:
good idea that will make this a bit cleaner